app/models/user.rb

```diff
@@ -1,4 +1,10 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :confirmable
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/209318681-83198b0c-c851-4aae-a341-e2d2f82151ac.png)

- [ ] When i run rails server then this error popped up on my screen which you can see in the image above :arrow_up: Kindly fix this issue of tailwind :+1:

Reply:
But it is working just fine on my side, 
I am sorry to take this long to respond